lib/tty/commands/add.rb

```diff
@@ -272,6 +274,14 @@ def cmd_constantinized_parts
         ].compact
       end
 
+      # Make sure the current directory is a teletype project
+      # (or, at the very least, contains the correct directory structure)
+      #
+      def validate_pwd
+        fail ::TTY::CLI::Error, @pastel.red("This doesn't look like a teletype app directory - are you in the right place?") unless
+          options[:force] || (@app_path + "lib/#{namespaced_path}").exist?
+      end
```

Comment:
Since we add commands to `lib/app/commands` folder, maybe we should check that this location is present? After displaying the error message, we could potentially advise on how to run the command with an example or similar to Rails automatically run `teletype add --help` ?

Reply:
I originally was checking that, yes - but it was causing other tests to fail (they set up a fixture that did not make a 'commands' dir). The add command creates the 'commands' directory if it doesn't exist (seemingly):

```
[katelyn@h test]$ rm -r lib/test/commands
[katelyn@h test]$ teletype add haha
      create  spec/integration/haha_spec.rb
      create  spec/unit/haha_spec.rb
      create  lib/test/commands/haha.rb
      create  lib/test/templates/haha/.gitkeep
      inject  lib/test/cli.rb
[katelyn@h test]$ ls -l lib/test/commands
total 4
-rw-r--r-- 1 katelyn katelyn 332 Sep 28 15:07 haha.rb
```
..so I don't think the check is necessary.

I like the idea of outputting the 'help' - yeah. I'll add that!